ddtrace/contrib/aioredis/patch.py

```diff
@@ -175,7 +175,7 @@ def _finish_span(future):
             redis_command = span.resource.split(" ")[0]
             future.result()
```

Comment:
Do we need this first call if we already do it in the next call to `determine_row_count()`?

Reply:
Don't know, haven't really thought about it.